tests/nested/core20/tpm/task.yaml

```diff
@@ -24,7 +12,7 @@ execute: |
     execute_remote "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    retry -n 5 --wait 5 execute_remote "test-snapd-mokutil --sb-state" | MATCH "SecureBoot enabled"
+    execute_remote "xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2 -aa0d-00e098032b8c" | MATCH "00000000: 0600 0000 01\s+....."
```

Comment:
I think there's a space in the middle of the file name that shouldn't be there (after 11d2)

Reply:
And maybe the MATCH regexp should be between single quotes so the backslash-s won't be expanded by the shell?

Reply:
fixed